README.md

```diff
@@ -1 +1,171 @@
-# Catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h2><b>Catalog of my things</b></h2>
+  <h4>Ivan Gonzalez & Ismael Mastronardi & Hajnalka Oltyan</h4>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🎥 Video Presentation](#video-presentation)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+**Catalog of my things** is a Ruby app that helps the user to keep a record of different types of things the user own: books, music albums, and games.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add books, music albums, and games**
+- **List the book, music album and game items**
+- **List the categories genres, labels and authors**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Video Presentation <a name="video-presentation"></a>
+
+ The link to the project presentation video:
+
+- [Project presentation video](https://github.com/ivangonzalez224/Catalog-of-my-things/assets/128931752/ee63c2f2-ebbb-452e-8363-6f14b6fc04ca)
+
```

Comment:
- [ ] Good job adding a video presentation. However, let's keep the maximum length of `5min` as it is [required](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=maximum%20length%20of%205%20minutes.) inside of 5.45min 👍.

![Screen](https://github.com/ivangonzalez224/Catalog-of-my-things/assets/57408419/d9d38203-f752-4a71-ac2e-cb37eb6da23b)

Reply:
Fixed.